app/Http/Requests/UpdateRegulatedOrganizationRequest.php

```diff
@@ -117,15 +117,22 @@ public function withValidator($validator)
     public function attributes(): array
     {
         return [
+            'name' => __('organization name'),
             'locality' => __('city or town'),
             'region' => __('province or territory'),
-            'contact_person_email' => __('email address'),
-            'contact_person_phone' => __('phone number'),
```

Comment:
Please review Figma and/or issues to see if this was a specific design requirement.

Reply:
Oh, I see what you meant by the previous comment. I did change these to the translations we have that suits the context. There was no specific design requirement about these, so I will revert back to what it was.